Sources/SwiftSyntaxMacros/MacroExpansionContext.swift

```diff
@@ -12,6 +12,7 @@
 
 import SwiftDiagnostics
 import SwiftSyntax
+@_spi(Diagnostics) import SwiftSyntaxBuilder
```

Comment:
I don’t think we need `@_spi(Diagnostics)` anymore.

Reply:
Good catch! Sorry I missed it.